deployment/Dockerfile

```diff
@@ -0,0 +1,21 @@
+##### BASE IMAGE #####
+FROM elixircloud/foca:20240711-py3.12
+
+##### METADATA ##### 
+LABEL software="Cloud Storage Handler"
+LABEL software.description="API for handling files on a cloud storage service"
+LABEL software.website="https://github.com/elixir-cloud-aai/tus-storagehandler"
+LABEL maintainer.organisation="ELIXIR Cloud & AAI"
+
+## Copy app files
+COPY ./tus_storage_handler /app
+
+## Install app
+RUN cd /app \
+  && poetry install \
+  && poetry shell \
+  && python app.py    
+
+WORKDIR /app
+
+CMD ["python", "tus_storage_handler/app.py"]
```

Comment:
If you are using console script for poetry why type the whole thing, ie `python tus_storagehandler/app.py` you can just use `tus_storagehandler`. But as I can see that you have remove the `main.py` file, I mean its opinionated what you wanna name it but if so then change the path of the console script from `tus_storagehandler.main:main` to `tus_storagehandler.app:main`.